packages/react/src/components/tabs/Tabs.tsx

```diff
@@ -71,3 +74,7 @@ export const Tabs = ({ children, small = false, theme }: TabsProps) => {
     </TabsContext.Provider>
   );
 };
+
+Tabs.TabList = TabList;
```

Comment:
I guess the burning question is that will is break anything in projects that currently use the Tabs component?

Reply:
No, I did not change the exports, they can still use them as separate components. It is not the preferred way though. I'll fix the examples as well

Reply:
I'll fix the examples

Reply:
Maybe add a comment here to explain why this was done? Although might be hard if it is not completely certain why this fixes the issue.

Reply:
I just updated the description. Comment does it make sense 😁

Reply:
I mean adding a comment to before this actual code. I don't know how easy it would be to find this PR afterwards when someone starts wondering why this was done.

Reply:
Yes. I could add that.

Reply:
- [x] add code comment